src/parser/shared/modules/features/SelfHealTimingGraph.js

```diff
@@ -80,15 +80,15 @@ class SelfHealTimingGraph extends Analyzer {
       });
 
     const _casts = this._selfhealTimestamps.map(event => {
-      const p = (event.hitPoints / event.maxHitPoints) || 0;
+      const p = ((event.hitPoints - (event.amount || 0) + (event.absorb || 0)) / event.maxHitPoints) || 0;
```

Comment:
since you re-use this formula, should probably do like

```js
const startingHP = event.hitPoints - (event.amount || 0) + (event.absorb || 0);
const p = startingHP / event.maxHitPoints;
```
and then re-use `startingHP` on line 91